packages/components/organisms/f-offers/src/store/offers.module.js

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios';
+import jwtDecode from 'jwt-decode';
+import {
+    MUTATION_SET_AUTH_TOKEN,
+    MUTATION_SET_BRAZE_KEY,
+    MUTATION_SET_CUSTOMER_DATA,
+    ACTION_INITIALISE_OFFERS,
+    ACTION_HANDLE_CUSTOMER_DATA
+} from './types';
+
+/**
+ * Setup the config for axios
+ * @param authHeader
+ * @param timeout
+ * @returns {{headers: {Authorization, "Content-Type": string}, timeout}}
+ */
+const createAxiosConfig = (authHeader, timeout) => ({
+    headers: {
+        'Content-Type': 'application/json',
+        Authorization: authHeader
+    },
+    timeout
+});
+
+/**
+ * @function resolveCustomerDetails
+ * @param  {object} data  - Api response object.
+ * @param  {string} authToken - auth token.
+ */
+const resolveCustomerDetails = (data, authToken) => {
+    const tokenData = jwtDecode(authToken);
+    return {
+        phoneNumber: data?.customer?.phoneNumber ? data?.customer?.phoneNumber : tokenData?.phone_number,
+        firstName: data?.customer?.firstName ? data?.customer?.firstName : tokenData?.given_name,
+        lastName: data?.customer?.lastName ? data?.customer?.lastName : tokenData?.family_name
+    };
+};
+
+export default {
+    namespaced: true,
+
+    state: () => ({
+        authToken: 'test',
+
+        customer: {
+            friendlyName: 'Matthew',
+            firstName: '',
+            lastName: ''
+        },
+
+        brazeApiKey: undefined
+    }),
+
+    getters: {
+        isAuthenticated: state => !!state.authToken,
+        friendlyName: state => state.customer.friendlyName
+    },
```

Comment:
Please can the order of keys in stores be reordered to `state`, `actions`, `mutations` and then `getters`? Currently we haven't yet defined any getters within stores in this repo, but in several internal consumer-facing components, this is the order that is used, so it would be good to be consistent here.